packages/react/src/components/Dashboard/DashboardGrid.test.e2e.jsx

```diff
@@ -1158,6 +1158,93 @@ DashboardAllCardsAsResizable.propTypes = {
   type: PropTypes.oneOf(CARD_TYPES).isRequired,
 };
 
+describe('DashboardGrid-lg-Card', () => {
+  beforeEach(() => {
+    MockDate.set(1537538254000);
+    cy.viewport(1024, 900);
+  });
+  afterEach(() => {
+    MockDate.reset();
+  });
+
+  it('Card with range picker', () => {
```

Comment:
@davidicus I changed the test name. Also, I updated acceptance tests because now closing dropdown by click outside will restore previously saved value.